gbpservice/nfp/core/controller.py

```diff
@@ -575,6 +575,9 @@ def common_init():
     oslo_config.CONF.register_opts(nfp_config.OPTS)
     oslo_config.CONF.register_opts(
         nfp_config.es_openstack_opts, "keystone_authtoken")
+    oslo_config.CONF.set_override('use_syslog', 'True')
```

Comment:
Can we add a comment as following.

Since other imports are registering the logging configuration parameters, these are overridden to make sure core configuration parameters are effective.